### PR TITLE
fixed issue. added metric of leaves number

### PR DESCRIPTION
**Pull request by Vitaly-Protasov**
fixing #196

### CODE REVIEW

File: aibolit/metrics/countLeaves/numberofleaves.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    def __init__(self):
+        pass
+
+    def value(self, filename: str):
+
+        tree = JavalangImproved(filename)
+
+        nodes = tree.tree_to_nodes()
+        traversed = []
+        for each_noda in nodes:
```
Comment (acheshkov) [inline aibolit/metrics/countLeaves/numberofleaves.py:21]: `each_node`

File: aibolit/metrics/countLeaves/numberofleaves.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+class CountNumberOfLeaves:
+    '''
+    Returns number of leaves for each method in class.
+    input: file_path
+    output: list of numbers. Size of this list == number of methods in class.
+    '''
+    def __init__(self):
+        pass
+
+    def value(self, filename: str):
```
Comment (acheshkov) [inline aibolit/metrics/countLeaves/numberofleaves.py:15]: Metric has to return a number, not a list of lines. 
```
def value(self, filename: str) -> float:
```

**Comment by acheshkov** [comment]
@Vitaly-Protasov test the PR with `flake8`

**Comment by acheshkov** [comment]
@0crat out

File: aibolit/metrics/countLeaves/numberofleaves.py (hunk 1 above)
Comment (Vitaly-Protasov) [reply aibolit/metrics/countLeaves/numberofleaves.py:21]: @acheshkov  fixed

**Comment by Vitaly-Protasov** [comment]
> @Vitaly-Protasov test the PR with `flake8`

@acheshkov  tested. it's fine

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/198#issuecomment-612037662))

@acheshkov Job `gh:yegor256/aibolit#198` is not assigned, can't get performer

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/198#issuecomment-612037662))

@acheshkov Pull request [#198](https://github.com/yegor256/aibolit/issues/198) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

File: aibolit/metrics/countLeaves/numberofleaves.py (hunk 2 above)
Comment (Vitaly-Protasov) [reply aibolit/metrics/countLeaves/numberofleaves.py:15]: > Metric has to return a number, not a list of lines.
> 
> ```
> def value(self, filename: str) -> float:
> ```

fixed

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#198` is not assigned, can't get performer

**Pull request merged**